src/main/java/no/ssb/dlp/pseudo/service/sid/SidMapper.java

```diff
@@ -116,8 +118,13 @@ private PseudoFuncOutput createMappingLogsAndOutput(SidInfo sidInfo, boolean isF
                 PseudoFuncOutput output = PseudoFuncOutput.of(sidInfo.snr());
                 output.addWarning(message);
                 return output;
+            }else{
```

Comment:
```suggestion
            } else {
```
The code is not properly formatted. The IDE should fix that for you

Reply:
fixed